tests/next_tests/integration_tests/multi_feature_tests/iterator_tests/test_with_toy_connectivity.py

```diff
@@ -205,8 +215,10 @@ def test_sparse_input_field(program_processor, lift_mode):
         assert np.allclose(out, ref)
 
 
-def test_sparse_input_field_v2v(program_processor, lift_mode):
-    program_processor, validate = program_processor
+def test_sparse_input_field_v2v(program_processor_no_dace_exec, lift_mode):
+    program_processor, validate = program_processor_no_dace_exec
+    if program_processor == run_dace_iterator:
+        pytest.xfail("Not supported in DaCe backend: assert fails")
```

Comment:
Can we have a more meaningful error messasge here? Judging from the name, it is probably due to "sparse fields". (And other reasons possibly as well)

Reply:
In this case I can just rely on the dace fixture, and remove the additional error message.